components/bl-video-recorder-component/src/helpers.js

```diff
@@ -0,0 +1,44 @@
+export const prepareToRecord = videoRef => {
+  if (videoRef) {
+    videoRef.autoplay = true;
+    videoRef.muted = true;
+    videoRef.controls = false;
+  }
+};
+
+export const prepareToView = videoRef => {
+  if (videoRef) {
+    videoRef.autoplay = false;
+    videoRef.muted = false;
+    videoRef.controls = true;
+    videoRef.srcObject = null;
+    videoRef.captureStream = null;
+  }
+};
+
+export const download = (blob, fileName) => {
+  const link = document.createElement('a');
+  link.style.display = 'none';
+  link.href = URL.createObjectURL(blob);
+  link.download = fileName;
+
+  document.body.appendChild(link);
+  link.click();
+
+  setTimeout(() => {
```

Comment:
Why we use timeouts here?

Reply:
To make this work on Firefox we need to wait a little while before removing the link.